src/modules/Bootloader.rb

```diff
@@ -156,9 +156,20 @@ def Read
         ::Bootloader::BootloaderFactory.current.read
       rescue ::Bootloader::UnsupportedBootloader => e
         ret = Yast::Report.AnyQuestion(_("Unsupported Bootloader"),
-          _("Unsupported bootloader '%s' detected. Show proposal of supported configuration?") %
+          _("Unsupported bootloader '%s' detected. Use proposal of supported configuration instead?") %
             e.bootloader_name,
-          _("Show"),
+          _("Use"),
+          _("Quit"),
+          :yes) # focus proposing new one
+        return false unless ret
+
+        ::Bootloader::BootloaderFactory.current = ::Bootloader::BootloaderFactory.proposed
+        ::Bootloader::BootloaderFactory.current.propose
+      rescue ::Bootloader::BrokenConfiguration => e
+        ret = Yast::Report.AnyQuestion(_("Broken Configuration"),
+          _("Bootloader configuration is broken (%s). Propose configuration from scratch?") %
```

Comment:
Maybe we should be less assertive about current configuration. What about "YaST cannot process current bootloader configuration (%s). Propose new configuration from scratch?". Even if we can't parse it/understand it, maybe is not broken strictly speaking.